pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -2621,4 +2621,6 @@ self: super: {
   # compatability with other gi- packages.
   # Take another look when gi-webkit2 updates as it may have become compatible with libsoup-3
   gi-soup = assert versions.major self.gi-webkit2.version == "4"; self.gi-soup_2_4_28;
+
+  llvm-ffi = super.llvm-ffi.override {LLVM = pkgs.llvmPackages_13.libllvm;};
```

Comment:
Does it more sense to use the default LLVM (i.e. `self.llvmPackages.libllvm`) used for the compiler (maybe just to reduce closure size in some cases)? Seems like llvm-ffi supports multiple versions.